VMAccess/vmaccess.py

```diff
@@ -324,7 +331,9 @@ def _reset_sshd_config(sshd_file_path):
                     break
 
         # Prepare cloud init config for coreos-cloudinit
-        cfg_tempfile = "/tmp/cloudinit.cfg"
+        f = tempfile.NamedTemporaryFile(delete=False)
+        f.close()
+        cfg_tempfile = f.name
```

Comment:
I do not see the value in piping this construct through STDIN.  The requirement is to **not** use a guessable temporary file, which is what this change fixes.  There are no protected settings exposed in this configuration, the intent is only to prevent others from modifying the file before application.